File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        
        int n = nums.size();

        int nSum = n*(n+1);
        nSum = nSum/2;

        int aSum = 0;

        for(int i =0; i< n ; i++){

            aSum += nums[i];
        }

        int ans = nSum - aSum;

        return ans;
    }
};

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        
        int n = nums.length;

        int totalSum = n*(n+1)/2;

        int arraySum =0;

        for(int i : nums){
            arraySum += i;
        } 

        return totalSum - arraySum;
    }
}